cli/pcluster/utils.py

```diff
@@ -1112,3 +1112,30 @@ def get_bucket_url(region):
 def get_file_section_name(section_key, section_label=None):
     """Build a section name as in the config file, given section key and label."""
     return section_key + (" {0}".format(section_label) if section_label else "")
+
+
+def get_ebs_snapshot_info(ebs_snapshot_id):
+    """
+    Return a dict described the information of an EBS snapshot returned by EC2's DescribeSnapshots API.
+
+    Example of output:
+    {
+        "Description": "This is my snapshot",
+        "Encrypted": False,
+        "VolumeId": "vol-049df61146c4d7901",
+        "State": "completed",
+        "VolumeSize": 120,
+        "StartTime": "2014-02-28T21:28:32.000Z",
+        "Progress": "100%",
+        "OwnerId": "012345678910",
+        "SnapshotId": "snap-1234567890abcdef0",
+    }
+    """
+    try:
+        return boto3.client("ec2").describe_snapshots(SnapshotIds=[ebs_snapshot_id]).get("Snapshots")[0]
+    except ClientError as e:
+        error(
+            "Failed when calling DescribeSnapshot for {0}: {1}".format(
+                ebs_snapshot_id, e.response.get("Error").get("Message")
+            )
+        )
```

Comment:
let's add tests for this function. they can be added to the files in cli/tests/pcluster/utils/test_pcluster_utils.py

Reply:
Yes! It is done